apps/fortran/airfoil/airfoil_arg_ptrs/dp/airfoil.F90

```diff
@@ -36,8 +36,18 @@ program AIRFOIL
   type(op_dat) :: p_bound, p_x, p_q, p_qold, p_adt, p_res
 
   ! arrays used in data
-  integer(4), dimension(:), allocatable, target :: ecell, bound, edge, bedge, becell, cell
-  real(8), dimension(:), allocatable, target :: x, q, qold, adt, res, q_part
+  !integer(4), dimension(:), allocatable, target :: ecell, bound, edge, bedge, becell, cell
+  real*8 x, q, qold, adt, res, q_part
```

Comment:
Hydra defines these as 1-long arrays instead of scalars. Not sure if it has significance, but I think we might get warnings about passing scalars to dummy array arguments.

Reply:
ok done